Source/Scene/Expression.js

```diff
@@ -1060,8 +1104,17 @@ define([
             return Cartesian3.add(left, right, ScratchStorage.getCartesian3());
         } else if ((right instanceof Cartesian4) && (left instanceof Cartesian4)) {
             return Cartesian4.add(left, right, ScratchStorage.getCartesian4());
+        } else if ((typeof(left) === 'string') || (typeof(right) === 'string')) {
+            return left + right;
```

Comment:
This relies on each types `toString` function. Should the spec be explicit about how each type is stringified?